includes/news/mail/in-the-news.php

```diff
@@ -18,22 +18,39 @@
 					<ul style="list-style: none; padding-left: 0; margin: 0;">
 					<?php
 					foreach( $external as $story ) :
+						$story_source = ( !empty( $story->source_name ) ) ? $story->source_name : $story->source;
 					?>
 						<li style="padding-bottom: 30px;" align="left">
 							<table>
 								<tbody>
-								<tr>
-									<td style="padding-bottom: 4px;">
-										<a class="montserratsemibold" style="font-family: Helvetica, Arial, sans-serif; font-size: 18px; color: #000000; font-weight: 500;" href="<?php echo $story->url; ?>">
-											<?php echo $story->link_text; ?>
-										</a>
-									</td>
-								</tr>
-								<tr><td style="padding-top: 5px;">
-									<a class="montserratlight" style="font-family: Helvetica, Arial, sans-serif; font-size: 12px; color: #757575; text-decoration: none; text-transform: uppercase;" href="<?php echo $story->url; ?>">
-										<?php echo $story->source; ?>
-									</a>
-								</td></tr>
+									<tr>
+									<?php if( !empty( $story->source_image ) ) : ?>
+										<td style="padding-right: 10px;">
```

Comment:
We could increase this padding-right by 10px or so since we have quite a bit of spacing for the other elements in the email. Also, adding the `valign` attribute and `verticle-align: top` style makes sure that the cell/image is aligned with the top of the text in smaller sizes if the text cell has a higher height than the image:
```suggestion
										<td style="padding-right: 20px; vertical-align: top;" valign="top">
```